tunnistamo/tests/test_oidc_provider_pkce.py

```diff
@@ -11,7 +11,7 @@
 @pytest.mark.django_db
 @pytest.mark.parametrize('code_verifier,should_succeed', (
```

Comment:
I'd prefer to give them names instead of not using a random value. e.g.

```python
@pytest.mark.parametrize('code_verifier,should_succeed', (
    ('', False),
    (get_random_string(), True),
), ids=['empty_string', 'random_string'])
```

Reply:
Did that. Although with a slightly different syntax, only giving names for the random parameter tests and letting Pytest handle the rest.